### PR TITLE
Deprecated

### PR DESCRIPTION
**Pull request by bezoerb**

### CODE REVIEW

**Comment by addyosmani** [comment]
Reviewed. Thanks for working on this!

**Comment by bezoerb** [comment]
@addyosmani ping ;)

File: README.md [hunk 1]
```diff
@@ -109,6 +109,24 @@ critical.generate({
...
+
+Generate and return output via promise:
+
+```js
+critical.generate({
+    base: 'test/',
+    src: 'index.html',
+    width: 1300,
+    height: 900
+}).then(function(output) {
```
Comment (arthurvr) [inline README.md]: `function (output) {`

File: README.md [hunk 2]
```diff
@@ -132,38 +150,16 @@ critical.generate({
...
-critical.inline({
+critical.generate({
     base: 'test/',
-    src: 'index-critical.html'
-}, function (err, output){
-    // You now have HTML with inlined critical-path CSS
-    // Works with and without dest specified
+    src: 'index.html',
+    dest: 'styles/main.css',
+    ignore: ['@font-face',/(some|regexp)/,'.my-selector']
```
Comment (arthurvr) [inline README.md]: `ignore: ['@font-face', /(some|regexp)/, '.my-selector']`

**Comment by bezoerb** [comment]
@addyosmani so i guess you are fine with the changes ;)

**Comment by addyosmani** [comment]
🎩👏

**Comment by bezoerb** [comment]
\o/

**Pull request merged**